draft-ietf-mops-streaming-opcons.md

```diff
@@ -691,6 +696,8 @@ The media server may also choose to alter which bitrates are made available to p
 
 ## Adaptive Segmented Delivery {#adapt-deliver}
 
+Adaptive Segmented Delivery (ASD) attempts to optimize its own use of any given path. As noted in {{sec-band-constraints}}, video streams can encounter bottlenecks at many points along a path, whether the bottleneck happens at a node or along a path segment along the path, and these bottlenecks may involve processing power, buffering capacity, link speed, or any other exhaustible resource. The following description of ASD assumes that the path has previously been evaluated for bottlenecks - for instance, that any IEEE 802.11 wireless links along the path have been configured with appropriate QoS parameters {{IEEE802.11}}, which is typically out of the control of the media server (or even the media provider).
```

Comment:
I don't think the description assumes the path has been previously evaluated for bottlenecks, or that there are QoS parameters involved.  The point of the adaptation is that it adapts to whatever the conditions are, even if it has not been previously evaluated.

Reply:
So maybe just cut the last sentence and I think the rest is mostly ok?

Reply:
But while we're at it, also "along a path segment along the path" parses odd.  Maybe "at a path segment along the delivery path"?

Anyway, I think if the point is to include a mention of QoS in response to issue #182, I'm not sure it goes here.  I think it's more relevant for walled-garden deployments that don't need ABR because they have a known amount of reserved priority space, as far as I'm aware?  I've not heard of OTT ABR using or relying on QoS, since it's often not propagated and generally doesn't come with properties that can be relied on for internet traffic.

Reply:
@GrumpyOldTroll - I think you're right - most of this is more appropriate in the scoping section, because this document doesn't make recommendations about QoS, etc. I'll go there instead.